gen3/bin/kube-setup-workvm.sh

```diff
@@ -299,7 +299,7 @@ EOF
 
   if ! grep 'GEN3_HOME=' ${WORKSPACE}/.${RC_FILE} > /dev/null; then
     cat - >>${WORKSPACE}/.${RC_FILE} <<EOF
-export GEN3_HOME=${WORKSPACE}/cloud-automation
+export GEN3_HOME=${GEN3_HOME}
```

Comment:
I think we still need to set this variable, or it will never be set.

Reply:
GEN3_HOME is correctly calculated based on location of kube-setup-workvm.sh in the beginning of the script:
https://github.com/uc-cdis/cloud-automation/blob/5715aa4ff7d98184db88ee54f27f92e7305d6f2d/gen3/bin/kube-setup-workvm.sh#L11

Thus there shouldn't be a need to force it into ${WORKSPACE}/cloud-automation